infracost-usage-example.yml

```diff
@@ -1185,3 +1185,25 @@ resource_usage:
   ibm_is_vpn_gateway.testVpcVpn:
     connection_hours: 730 # Monthly number of connection hours
     instance_hours: 730 # Monthly number of instance hours
+  #
+  # Cloud Object Storage
+  #
+  ibm_cos_bucket.standard-us-south:
```

Comment:
can these 5 blocks be collapsed into 1 block that contains all of the different usage variables?